.storybook/manager-head.html

```diff
@@ -0,0 +1,34 @@
+<style>
+  .sidebar-item {
+    padding-bottom: 6px !important;
+    padding-top: 6px !important;
+  }
+
+  .sidebar-item svg {
```

Comment:
```suggestion
  .sidebar-item.sidebar-item svg {
```

Reply:
You can likely remove all the uses of `!important` if you double up on the class name to raise the specificity.

Eg: `.sidebar-item.sidebar-item[data-selected="true"]`

Reply:
I was trying a few things and it wouldn't take, but I can check that again.